contrib/obfuscate_project

```diff
@@ -10,9 +10,12 @@ CONFIGURATION=Release
 SDK=7.1
 
 # Additional build options
-XCTOOL_OPTS=""
+XCODEBUILD_OPTS=""
 CLASS_GUARD_OPTS="-i IgnoredSymbol -F !ExcludedClass"
 
+# In case of using Xcode >= 6 and SDK >= 8
+CLASS_GUARD_OPTS_SDK="--sdk-root /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator$SDK.sdk"
```

Comment:
Using `xcode-select -p` causes issues with passing non-standard paths (e.g. with whitespace) in further part of script.